KomaMRICore/test/runtests.jl

```diff
@@ -1,6 +1,27 @@
 using TestItems, TestItemRunner
 
-@run_package_tests filter=ti->!(:skipci in ti.tags)&&(:core in ti.tags) #verbose=true
+### NOTE: by default, tests are run on the CPU with the number of threads set to 
+#   Threads.nthreads(). To run on a specific GPU backend, pass the name of the 
+#   trigger package ("AMDGPU", "CUDA", "Metal", or "oneAPI") as a test argument. 
+#   
+#   Example:
+#
+#   import Pkg
+#   Pkg.test("KomaMRICore"; test_args=["CUDA"])
+#
+#   To run on the cpu with a specific number of threads, pass the number of threads 
+#   as a julia argument.
+#
+#   Example:
+#   
+#   import Pkg
+#   Pkg.test("KomaMRICore"; julia_args=`--threads=4`)
+###
+
+#Environment variable set by CI
+const CI = get(ENV, "CI", nothing)
+
+@run_package_tests filter=ti->(:core in ti.tags)&&(isnothing(CI) || :skipci ∉ ti.tags) #verbose=true
```

Comment:
```suggestion
is_local_run = isnothing(CI) 
is_buildkite_run = CI == "BUILDKITE"

@run_package_tests filter=ti->(:core in ti.tags) && (is_local_run || :skipci ∉ ti.tags) && (!is_buildkite_run || :skipbuildkite ∉ ti.tags)
```

The logic might me wrong :sweat_smile:.